src/goalServices/reduceGoals.ts

```diff
@@ -351,6 +352,54 @@ function reducePrompts(
     }, promptsToReduce);
 }
 
+function reducePromptsForReview(
+  recipientId: number,
+  fullGrantName: string,
+  newPrompts:IPrompt[] = [],
+  previousPrompts:IReviewPrompt[] = [],
+) {
+  const updatedPrompts = [...previousPrompts];
+  newPrompts.forEach((currentPrompt) => {
+    // Get the prompt Id.
+    const promptId = currentPrompt.promptId
+      ? currentPrompt.promptId : currentPrompt.dataValues.promptId;
+
+    // Get the responses for the prompt sorted alphabetically.
+    const { response } = currentPrompt;
```

Comment:
Why isn't there the same .dataValues check here?

Reply:
Ill add a check.